packet/conn.go

```diff
@@ -75,11 +49,9 @@ func NewConn(conn net.Conn) *Conn {
 	c := new(Conn)
 	c.Conn = conn
 
-	c.bufPool = NewBufPool()
-	c.br = bufio.NewReaderSize(c, 65536) // 64kb
-	c.reader = c.br
+	c.reader = c
```

Comment:
ok so not sure about this, but the NewTLSConn doesn't create a buffered reader for this, so I mirrored that here as well. If there is a reason we need the buffered reader here and I can restore it with some comments for future reference.